pages/web-components.md

```diff
@@ -1,54 +0,0 @@
----
```

Comment:
This page hasn't been updated in years, reads in first person, and has outdated takes in it. If we want to recommend a certain web component ideology, we need to do it fresh.